File: SmileEditor/Model/Legacy/AreaSave.cs
namespace SmileEditor.Model.Legacy
{
    internal class AreaSave
    {
        internal int RoomID { get; set; }       // (Legacy)     RoomLoad As Integer 'points to door data that uses this RoomID
        internal int Unknown { get; set; }      // (Legacy) 	always zero??
        internal int ScrollX { get; set; }      // (Legacy)     ScrollY As Integer
        internal int SamusY { get; set; }       // (Legacy) 	as offset from top
        internal int SamusX { get; set; }    	// (Legacy) 	as offset from center

        internal AreaSave() { }
    }
}

File: SmileEditorLegacy/Parsed/Types/Model/AreaSave.vb
Public Type AreaSave
    RoomID As Integer
    RoomLoad As Integer 'points to door data that uses this RoomID
    Unknown As Integer  'always zero??
    ScrollX As Integer
    ScrollY As Integer
    SamusY As Integer   'as offset from top
    SamusX As Integer   'as offset from center
End Type
